src/logPoints/LogPointsDebugAdapter.ts

```diff
@@ -8,8 +8,8 @@ import {
     DebugSession, Event, InitializedEvent, Logger, logger,
     LoggingDebugSession, Source,
     TerminatedEvent, Thread
-} from 'vscode-debugadapter';
-import { DebugProtocol } from 'vscode-debugprotocol';
+} from 'vscode-debugadapter/lib/main';
+import { DebugProtocol } from 'vscode-debugprotocol/lib/debugProtocol';
```

Comment:
I'm not sure where these and a few other of the imports got changed from...

Reply:
Well can ya change it back?